siglens/siglens
# Fix: SQL Query Limit Issue

# Description
This PR addresses a bug where the SQL query with LIMIT  was ignoring the limit and displaying all results. The changes ensure that only the specified number of results are returned.

Fixes #201 

# Testing
- Run an SQL query with a specified limit. For example, `SELECT * FROM * LIMIT 5`. Verify that only 5 results are returned.
- Run an SQL query without any limit and verify that all records are returned.
- Run a groupby request with a specified limit. For example, `SELECT COUNT(*), column FROM table GROUP BY column LIMIT 2`. Verify that only 2 groups are returned.
![image](https://github.com/siglens/siglens/assets/113487319/5be131dc-23d2-435b-96c2-0171d25be9e3)


# Checklist:
Before marking your pull request as ready for review, complete the following.
